File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/executing/obfuscate/RedactingPHIObfuscator.java
package org.opencds.cqf.cql.engine.elm.executing.obfuscate;


public class RedactingPHIObfuscator implements PHIObfuscator {

    public static final String REDACTED_MESSAGE = "<redacted>";

    @Override
    public String obfuscate(Object source) {
        return REDACTED_MESSAGE;
    }

}

File: Src/java/engine/src/commonMain/kotlin/org/opencds/cqf/cql/engine/elm/executing/obfuscate/RedactingPHIObfuscator.kt
package org.opencds.cqf.cql.engine.elm.executing.obfuscate

import org.opencds.cqf.cql.engine.runtime.Value

class RedactingPHIObfuscator : PHIObfuscator {
    override fun obfuscate(source: Value?): String {
        return REDACTED_MESSAGE
    }

    companion object {
        const val REDACTED_MESSAGE: String = "<redacted>"
    }
}
